### PR TITLE
adapt report data to ui objects

### PR DESCRIPTION
**Pull request by swrdfish**

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
     <Wrapper>
       <Left>
         <NightwatchIcon />
       </Left>
       <Center>Test Reporter</Center>
       <Right>
-        <Date>{getDateFromMillisecond(date)}</Date>
-        <Time>{convertMsToTime(timestamp).paddedTime}</Time>
+        <DateComponent>{getDateFromMillisecond(parsedDate)}</DateComponent>
+        <Time>{parsedDate.toLocaleDateString ()}</Time>
```
Comment (vaibhavsingh97) [inline src/components/Header/index.tsx:22]:
```suggestion
        <Time>{parsedDate.toLocaleTimeString()}</Time>
```

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import { useGlobalContext } from '../../hooks/GlobalContext';
 import { NightwatchIcon } from '../../icons';
-import { convertMsToTime } from '../../utils';
-import { Center, Date, Left, Right, Time, Wrapper } from './style';
+import { Center, Date as DateComponent, Left, Right, Time, Wrapper } from './style';
```
Comment (vaibhavsingh97) [inline src/components/Header/index.tsx:4]: We don't need to rename `Date` as react infer component only when it's start as capital letter

File: src/components/Header/util.tsx [hunk 3]
```diff
@@ -1,5 +1,4 @@
-export const getDateFromMillisecond = (milliseconds: number) => {
-  const date = new Date(milliseconds);
+export const getDateFromMillisecond = (date: Date) => {
```
Comment (vaibhavsingh97) [inline src/components/Header/util.tsx:1]: Can we fix the function name from `getDateFromMillisecond` to `getParsedDate` or something more generic?

File: src/components/Summary/Analytics.tsx [hunk 4]
```diff
@@ -46,7 +47,7 @@ const Analytics: React.FC = () => {
                 Tests
               </Text>
             </TotalSpecCountWrapper>
-            <Timer time={time} color={'--color-grey-90'} />
+            <Timer time={convertMsToTime(time).time} color={'--color-grey-90'} />
```
Comment (vaibhavsingh97) [inline src/components/Summary/Analytics.tsx:50]: Please destruct it and use the time here

```
const {time} = convertMsToTime(time);
```

File: src/components/TestStatusBar/index.tsx [hunk 5]
```diff
@@ -10,7 +10,7 @@ const TestStatusBar: React.FC = () => {
   const { environmentName } = useReportContext();
   const { environments } = useGlobalContext();
 
-  const { time } = environments[environmentName].metadata;
+  const { time } = environments[environmentName].stats;
```
Comment (vaibhavsingh97) [inline src/components/TestStatusBar/index.tsx:13]: same, destruct stats, and then time

```
  const { stats: {time} } = environments[environmentName].stats;
```

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: src/components/Header/index.tsx (hunk 2 above)
Comment (swrdfish) [reply src/components/Header/index.tsx:4]: Date component was conflicting with the Date object. did not want to make another util function just to parse date.

**Pull request merged**